bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java

```diff
@@ -228,6 +231,47 @@ public static <T> List<T> copyList(@Nullable Collection<T> theList) {
         }
     }
 
+    @NotNull
+    public static <T> List<T> deepCopy(@NotNull List<T> src) {
```

Comment:
I think a "Missing a Javadoc comment." has some meaning here.

Reply:
I believe the method name is already self-explanatory.